DOCTYPE and HTML tags count for two (not three) passing tests

### Canvas Link

https://learning.flatironschool.com/courses/5506/assignments/170653?module_item_id=371074

### Concern

Notes say:
If written correctly, running learn test now will pass three of the seven tests.

But I have added DOCTYPE, <head>, and <body> tags and 'learn test' output shows:


  index.html
    valid document structure
      ✔ has a DOCTYPE tag
      ✔ has opening and closing HTML tags
      1) has a language attribute in the <html> tag


  2 passing (7ms)
  1 failing

  1) index.html
       valid document structure
         has a language attribute in the <html> tag:
     AssertionError: expected html to have an attribute 'lang'

### Additional Context

_No response_

### Suggested Changes

? Should notes say:
If written correctly, running learn test now will pass two of the seven tests.

That is, the third test seems to be about attribute 'lang', not a test for <body>

--- a/README.md
+++ b/README.md
@@ -42,8 +42,14 @@ Let's start by adding the bare essentials of an HTML document to the provided
 > inside the `html` tags, if you haven't already.
 
 If written correctly, running `learn test` now will pass three of the seven
-tests. **However** you will not be done yet! We have to make **all** the tests
-pass in order to make the test code happy! Let's get to it!
+tests:
+
+- `has a DOCTYPE tag`
+- `has opening and closing HTML tags`
+- `has <head> and <body> tags nested in the <html> tag`
+
+However you will not be done yet! We have to make **all** the tests pass in
+order to make the test code happy! Let's get to it!
 
 Let's take a closer look at these tags.
 

--- a/index.html
+++ b/index.html
@@ -1,0 +1,10 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+
+  </head>
+  <body>
+
+  </body>
+</html>

